omod/src/main/resources/apps/ugandaemr_reports_app.json

```diff
@@ -102,6 +102,14 @@
         "url": "reportingui/runReport.page?reportDefinition=167cf668-071e-488b-b159-d5f391774038",
         "order": 10,
         "requiredPrivilege": "App: ugandaemrreports.HMIS106aSection1B"
+      },{
+        "id": "HMIS 106a Section 1B 2019",
+        "extensionPointId": "org.openmrs.module.ugandaemr.reports.quarterly2019",
```

Comment:
@dbaluku change the extension point to `org.openmrs.module.ugandaemr.reports.quarterly` as per this PR from Solomon to change how the reports are accessed https://github.com/METS-Programme/openmrs-module-ugandaemr-reports/pull/222

Reply:
Done